modules/apps/adaptive-media/adaptive-media-image-js-web/src/main/resources/META-INF/resources/ckeditor/plugins/adaptivemedia/plugin.js

```diff
@@ -35,7 +35,7 @@ function sourceTagTemplate({media, srcset}) {
 
 					let onSelectedImageChangeFn;
 
-					if (event.data.commandData) {
+					if (typeof event.data.commandData === 'function') {
```

Comment:
Lack of native events is marked as `{}`, not a falsy value. It looks like the original solution never applied `adaptivemedia`, or this changed internally to CKE with some base version update.